src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+import java.util.*;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+  public static void main(String[] args) {
```

Comment:
Should basic indentation be changed from 2 spaces to 4 spaces to align with the Java coding standard?

```suggestion
    public static void main(String[] args) {